test/slow/api_spec_slow.js

```diff
@@ -181,6 +191,10 @@ describe(`electron-forge API (with installer=${installer.substr(12)})`, () => {
           it('successfully makes with default config', async () => {
             await forge.make({ dir, skipPackage: true });
           });
+
+          it('successfully makes to outDir with default config', async () => {
```

Comment:
Didn't notice this at first, but this is going to dramatically increase test time right? 😢